tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlStoredProcedureTests.cs

```diff
@@ -105,24 +104,40 @@ public void Test()
             );
 
         }
-
-
     }
 
 
     [NetFrameworkTest]
-    public class MySqlStoredProcedureTestsWithAtSigns : MySqlStoredProcedureTestsBase
+    public class MySqlStoredProcedureTestsWithAtSignsFW : MySqlStoredProcedureTestsBase<ConsoleDynamicMethodFixtureFWLatest>
     {
-        public MySqlStoredProcedureTestsWithAtSigns(MySqlBasicMvcFixture fixture, ITestOutputHelper output)  : base(fixture, output, true)
+        public MySqlStoredProcedureTestsWithAtSignsFW(ConsoleDynamicMethodFixtureFWLatest fixture, ITestOutputHelper output) : base(fixture, output, true)
         {
 
         }
     }
 
     [NetFrameworkTest]
-    public class MySqlStoredProcedureTestsWithoutAtSigns : MySqlStoredProcedureTestsBase
+    public class MySqlStoredProcedureTestsWithoutAtSignsFW : MySqlStoredProcedureTestsBase<ConsoleDynamicMethodFixtureFWLatest>
```

Comment:
Same thing here, add more subclasses to cover the older runtimes/client versions.  Since there is also the "with/without @ signs" distinction, there should be twelve total variations.